bedrock/settings/__init__.py

```diff
@@ -258,6 +258,13 @@
 CSP_REPORT_ONLY = config("CSP_REPORT_ONLY", default="false", parser=bool)
 CSP_REPORT_URI = config("CSP_REPORT_URI", default="") or None
 
+# Report violations on a small proportion of requests
+CSP_REPORT_PERCENTAGE = config(
+    "CSP_REPORT_PERCENTAGE",
+    default="0",
+    parser=float,
+)
```

Comment:
Maybe include the expected range of the value here? `0.0 to 1.0`?

Reply:
IIRC this changes in `django-csp@4` from 0-1 to 0-100 so it'll be more obvious now.